### DISCUSSION TITLE
[聚合] 缝合怪主页！

### DISCUSSION THREAD

**Discussion by CreeperIsASpy** [Meloong-Git/PCL#4564] [自定义主页]
## 超级集成的新闻-快捷入口一体主页!
全新实用、紧凑外观
转为游戏时间紧张，没有时间停下看主页的人设计!
它甚至**不需要**你滑动你珍贵的滚轮!
![image](https://github.com/user-attachments/assets/277b6452-15b8-4424-95fc-1c963f87602f)
仓库地址:https://github.com/Ext1nguisher/PCLhomepage

**Comment by ArgonTeam** [comment]
虽然但是，这个mcnmsl中的nmsl能不能替换成其他词语。。
![image](https://github.com/user-attachments/assets/f0644802-deab-4ee8-81c4-351551ba5c94)

**Reply by CreeperIsASpy** [reply]
这是我的名字
到时候我会把它替换成"欢迎使用缝合怪主页"
谢谢提醒,忘记改了

**Comment by ArgonTeam** [comment]
没错又是我，我来挑刺了(bushi)(
![屏幕截图 2024-08-22 092406](https://github.com/user-attachments/assets/7dc0ab6e-12d7-4344-a4fa-2ffda2aa3ac9)

1. 您看看这个主页在低分辨率屏幕上的表现

![屏幕截图 2024-08-22 092353](https://github.com/user-attachments/assets/df50ca73-7e00-4552-88d5-4304cc1b2461)

2.宁看看这个图片显示的有点***

3.颜色有点过于鲜艳

4. 不是每个人都能访问github

5.善用 \current|服务器ip ，你有这个Minecraft版本不代表别人有

6. 你能说一下什么是划划水服务器吗 #2528
![image](https://github.com/user-attachments/assets/99a1a375-f6c0-4260-8222-13778e9628f2)

7. 图标显示不全
![image](https://github.com/user-attachments/assets/d09ce203-ee61-42e1-8d62-568f24a1e555)

8. 由于背景过于鲜艳，常用网站的文字请替换为白色 #FFFFFF

9. 我没有其他的想说了

**Reply by Ad-closeNN** [reply]
[https://github.com/Ext1nguisher/PCLhomepage/releases/download/v1.1.4/Custom.xaml](https://github.com/Ext1nguisher/PCLhomepage/releases/download/v1.1.4/Custom.xaml)
你咋拿旧版的主页呢...

> 图标显示不全

网络问题，不过倒是建议 Author 试试自己下载 favicon.ico 然后放到快速的地方（比如 GitCode Raw?）

<details><summary>By The Way</summary>
<p>
这不用滚动的想法是很不错的

那行字的字体为 Minecraft AE

![image](https://github.com/user-attachments/assets/a921ef7c-56c4-4efb-8c79-9fd08ef33def)

972x530
![image](https://github.com/user-attachments/assets/d77b8de3-cdaf-41d3-8c2c-6e46ca005c9f)
1512x1053
![image](https://github.com/user-attachments/assets/e7b61b6a-5f72-4ae3-ad70-32d881462f45)
</p>
</details>

**Comment by CreeperIsASpy** [comment]
有几点我要提醒大家(持续更新):
1. 请使用最新版本的主页
在我的仓库 Release 里找到最新版本，下载即可
针对这点，我已经在我的 https://ext1nguisher.github.io/Custom.xaml 同步最新文件，联网更新也可以
2. 请大家在使用这个主页之前，先看一下主页欢迎卡片右下角的 `i`，查看使用须知。
3. 针对下方 "服务器" 板块是否违反龙猫条例
我在选择常用服务器的时候是会尽量从 [MC 找服网](https://www.mczfw.cn/) 找到排名靠前的优质服务器。
并且我选择的其他服务器也属于较为中性，人尽皆知的，不会引起误会。
4. 针对上方欢迎栏目字体的问题
我本人尝试过上方栏目在没有字体的情况，与有字体的情况差别不大，
且能够良好适配，所以不存在兼容性问题。
5. 关于按钮，卡片中的渐变色太过显眼的问题
我之后会慢慢全部改成 `{DynamicResource ColorBrushX}`，缓解视觉冲突。
6. 关于版本无法正确启动问题的解决方法
"光影1.20.1" 和 "光影1.20.4" 的按钮后续会进行移除，因为其无法触发。
其他按钮的解决方法请见主页欢迎卡片右下角的 `i` (使用须知)，这里不再赘述。
7. 关于图标显示不全的问题
图标链接我已经全部改为国内，如果出问题，请耐心等待
常出问题的图标是下方的 "版本选择" 图标和 "常用网站" 中的 MC Wiki、MC 官网图标。
8. 关于链接在 GitHub，国内访问受限的问题
目前的计划是，先把链接放在 GitHub，看大众对于主页的反响如何。
如果反响不错，我会挂在 Gitee 中并使用 CDN 加速。
9. 关于新闻版块字体阅读体验较差的问题
新闻版块的字数较多，文字密集，如果分辨率低可能阅读体验会很差。
如果你发现字看起来非常难受，可以把 PCL 窗口拉大点~~(废话)~~
如果字体发生了重叠，请到我到仓库交pull request或加我的`QQ:3372728115`私信反馈
我也会把下方两个按钮的上 margin 调小，尽量优化大家的体验。
10. 关于功能与简单主页发生重复
简单主页是一个快捷入口的主页，在快捷入口方面想要做到不重复很难。
我会尽量添加独有功能的。新闻版块就是一个例子。
11. 关于wiki摘录板块协议相关问题
Wiki摘录板块以及这个主页采用了和Wiki本身一样的协议
已经说明是从MCwiki摘录的，并添加指向协议本体和Wiki的链接
~~我没有商业化还需要讲吗~~
12. 关于有那么多 Bug 为啥不修
Author 的学业最近非常紧急，并且处于一个非常关键的时期
我们的学校8月25日就开学了，并且开学之后没有很多使用电子时间的机会
也希望大家谅解我，私密马赛!
13. 关于新闻板块更新的问题
提前透露:我写了一个`CustomReloader`，你们可以把它下载到电脑的 PCL 文件夹，
运行它可以自动更新电脑上的 FHG 主页。
我在网页端也会更新。但是，如果你想看第一手的消息，还是建议你使用 CR !


谢谢您对于 FHG 主页的支持
祝您使用愉快!

**Reply by ArgonTeam** [reply]
对了
上线前记得添加下龙猫说的版本文件

**Reply by CreeperIsASpy** [reply]
对于版本文件我暂时不打算添加
因为我的主页更新频率非常高
但是如果后期实现了自动化重写我可能会加入版本文件

**Reply by hejiehao** [reply]
~~用 GitHub Action 给版本文件疯狂 +1~~

**Comment by CreeperIsASpy** [comment]
现在开始可以使用https://ext1nguisher.github.io/Custom.xaml 来加载主页了
谢谢支持!
@LTCatt 能不能看一下呢?

**Reply by Ad-closeNN** [reply]
> 如果反响不错，我会挂在 Gitee 中并使用 CDN 加速。

**Reply by CreeperIsASpy** [reply]
> 请不要把主页放在 GitHub Pages 上……

我后面会把主页迁徙到 gitee 上，具体见[提醒贴](https://github.com/Hex-Dragon/PCL2/discussions/4564#discussioncomment-10415651)

**Comment by LTCatt** [comment]
目前质量欠佳，暂时不考虑收录，抱歉（

**Reply by CreeperIsASpy** [reply]
嗯。。能不能说一下欠在哪？

**Reply by allMagicNB** [reply]
> 嗯。。能不能说一下欠在哪？

说白了就是比不过简单主页。

**Reply by CreeperIsASpy** [reply]
那哪里比不过？

**Reply by hejiehao** [reply]
使用的下载地址：https://raw.githubusercontent.com/Ext1nguisher/PCLhomepage/main/Custom.xaml

个人观点：

<details><summary>图片</summary>
<p>

![image](https://github.com/user-attachments/assets/2a9e10d1-b29a-4714-ba32-7ba79d54f8dd)

</p>
</details> 

上面的图片里被蓝色矩形框住的是简单主页已经有的功能

在 `服务器` 板块中 `启动光影 1.20.1` 和 `启动光影 1.20.4` 不应该放在服务器板块中，因为这两个不属于服务器

进入服务器以及启动版本选择中的 `Minecraft JE` 必须安装版本名为 `MostUsed` 的版本才可以正常启动

在窗口大小比较小的情况下字体过小，阅读体验较差

**Reply by CreeperIsASpy** [reply]
> 使用的下载地址：https://raw.githubusercontent.com/Ext1nguisher/PCLhomepage/main/Custom.xaml
> 
> 个人观点：
> 
> 图片
> ![image](https://private-user-images.githubusercontent.com/98140773/361327592-2a9e10d1-b29a-4714-ba32-7ba79d54f8dd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ2NTg2ODksIm5iZiI6MTcyNDY1ODM4OSwicGF0aCI6Ii85ODE0MDc3My8zNjEzMjc1OTItMmE5ZTEwZDEtYjI5YS00NzE0LWJhMzItN2JhNzlkNTRmOGRkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI2VDA3NDYyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk2N2YxZjc5MGVmYzM4OGUxNDgwOTFiOWYxZWRmNTg2YTdkNWVjNmMxNTcwMmY4YzM2NjkwMGZiZGUxNWNiZDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.VKylI-2uUx2P_rLUw7IYqJkN-SgwhRlFsXlnDuh6aSo)
> 
> 上面的图片里被蓝色矩形框住的是简单主页已经有的功能
> 
> 在 `服务器` 板块中 `启动光影 1.20.1` 和 `启动光影 1.20.4` 不应该放在服务器板块中，因为这两个不属于服务器
> 
> 进入服务器以及启动版本选择中的 `Minecraft JE` 必须安装版本名为 `MostUsed` 的版本才可以正常启动
> 
> 在窗口大小比较小的情况下字体过小，阅读体验较差

你说的所有问题，请见https://github.com/Hex-Dragon/PCL2/discussions/4564#discussioncomment-10415651
我说了多少遍了，先看提醒贴，先看提醒贴，就是不看。。。

**Reply by CreeperIsASpy** [reply]
你看看你圈出来的都是啥？
这些东西全都是快捷入口，**想不从简单主页抄都难**！
你做出来一个简单主页不包含的快捷按钮试试。。。

**Reply by hejiehao** [reply]
请将已知问题解决再申请

**Reply by CreeperIsASpy** [reply]
> 请将已知问题解决再申请

你说的那些是问题吗？那是特性！

**Reply by WForst-Breeze** [reply]
主页设计很不错呢，「已知问题」指的就是你的那个提醒贴中提及的问题，尝试将那些问题解决完后再尝试申请（最好附加解决后的截图），这样通过的概率会大一点的🤔

**Comment by CreeperIsASpy** [comment]
致所有给这个主页提意见的人

首先，谢谢你们的支持。
如果你要提意见，请先阅读[提醒贴](https://github.com/Hex-Dragon/PCL2/discussions/4564#discussioncomment-10415651)再在这里发帖。
请文明用语，谢谢，主页因为你们而完善！

**Comment by hejiehao** [comment]
比较有特色的是 `服务器` 板块，建议单独抽出来做一个 Minecraft 服务器推荐
~~虽然龙猫不一定同意~~

**Reply by CreeperIsASpy** [reply]
这个比较有特色？~~思路清奇啊~~
我认为最有特色的wiki摘录没人说，结果服务器版块有特色。。。啊？
不是好多都搞了这个，最后龙猫说不行就没了吗。。
自信点，别说不一定，是100%不会同意。

**Reply by allMagicNB** [reply]
Wiki 摘录未遵循 [CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/)，全剧终。

**Reply by CreeperIsASpy** [reply]
遵循了，请你先好好读一下

**Reply by CreeperIsASpy** [reply]
要求是你可以随意更改,引用，只要你做出来的产品遵循同样的协议
我做到了，我的主页协议也是CC BY-NC-SA 4.0

**Reply by allMagicNB** [reply]
> 遵循了，请你先好好读一下

未遵循，未标明指向 [CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/) 协议的链接。

**Reply by hejiehao** [reply]
> You must give [appropriate credit ](https://creativecommons.org/licenses/by-nc-sa/3.0/#ref-appropriate-credit), provide a link to the license, and [indicate if changes were made ](https://creativecommons.org/licenses/by-nc-sa/3.0/#ref-indicate-changes).

提供该许可证的链接

**Reply by CreeperIsASpy** [reply]
在上面啊？那个天平图标！

**Reply by allMagicNB** [reply]
> 在上面啊？那个天平图标！

那是你主页的，而非 Minecraft Wiki 的。

**Reply by CreeperIsASpy** [reply]
> > 在上面啊？那个天平图标！
> 
> 那是你主页的，而非 Minecraft Wiki 的。

？啥意思？

**Reply by CreeperIsASpy** [reply]
我的主页就是我制作出来的产品
我的产品和其引用的其它产品使用了同样的协议
我的产品没有商业化。
我没不遵守啊！

**Reply by CreeperIsASpy** [reply]
你刚刚说的要求是

> > 遵循了，请你先好好读一下
> 
> 未遵循，未标明指向 [CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/) 协议的链接。

要我标明指向协议的链接
结果现在变成要我指向wiki的链接了
到底要哪个？

**Reply by hejiehao** [reply]
两个应该都要
Minecraft Wiki 的链接属于 `give appropriate credit`，协议的链接属于 `provide a link to the license`
也就是说，你要在附近给出 https://zh.minecraft.wiki/w/%E6%B3%A5%E5%B7%B4 和 https://creativecommons.org/licenses/by-nc-sa/3.0/

**Reply by CreeperIsASpy** [reply]
好的，谢谢大佬提醒
~~我觉得人家wiki官方可能都懒得理我~~

**Reply by Deep-Dark-Forest** [reply]
要是做服务器推荐的话，建议用那些在官方服务器推荐列表（findmcserver.com)出现过的服务器，以免引起争议

**Reply by CreeperIsASpy** [reply]
mc找服网也不行？！

**Comment by CreeperIsASpy** [comment]
我有个新奇的想法
既然有人说功能比简单主页少
那我就做个全是按钮的主页好了

**Comment by CreeperIsASpy** [comment]
我想把按钮渐变色做成从`{DynamicResource ColorBrush4}`渐变到`{DynamicResource ColorBrush2}`,但是发现不能执行，出现错误
![image](https://github.com/user-attachments/assets/66d10f82-7885-45a1-a656-f395e59e8990)
但是我的语法没有问题，未知这是否是启动器 bug 。正在反馈中。
~~Bugs resolved:3/10~~

**Reply by CreeperIsASpy** [reply]
@LTCatt 看一下吧

**Reply by wyc-26** [reply]
> 从 `{DynamicResource ColorBrush4}` 渐变到 `{DynamicResource ColorBrush4}`

你这渐变了个寂寞啊……

**Reply by MoYuan-CN** [reply]
如果是这样的话，请使用纯色，而不是渐变色……
另外除非有特别重要的事情，不然不要 @ 龙猫。~~人家也是很忙的~~

**Reply by Light-Beacon** [reply]
笔刷和颜色是两个东西

**Reply by CreeperIsASpy** [reply]
eee，打错了，是从`{DynamicResource ColorBrush4}`到`{DynamicResource ColorBrush2}`

**Reply by CreeperIsASpy** [reply]
> 笔刷和颜色是两个东西

@Light-Beacon 细说

**Reply by MoYuan-CN** [reply]
笔刷是笔刷，颜色是颜色，反正就是两个东西，要求颜色的地方不能填笔刷

**Reply by CreeperIsASpy** [reply]
我把`{DynamicResource ColorBrush4}`换成了`{DynamicResource Color4}`，虽然不会出现错误了，但是颜色也没有了。。。

**Reply by CreeperIsASpy** [reply]
你们hooray干啥。。。。。没颜色了很难看的好吗(悩)
成这样了:
![image](https://github.com/user-attachments/assets/710fbde2-5b4f-4bad-98b1-cdf5f5517a30)

修复了以下Bug:

1. 新闻板块文本不可读问题
2. 按钮无法启动问题(学会怎么用\current了，以前不知道还有这玩意)

部分修复了以下Bug:
1. 新闻板块协议版权问题
2. 渐变色刺眼问题
3. 服务器板块引起误解(增加了提醒按钮)

感谢支持!
~~Bugs resolved:5/10~~

**Reply by hejiehao** [reply]
> 学会怎么用\current了，以前不知道还有这玩意

~~教学文件里有~~

**Reply by CreeperIsASpy** [reply]
我是这样的，从来不看教学文件
学任何东西都是这样的
~~学语文从不看课本~~

**Comment by CreeperIsASpy** [comment]
### 新版本更新日志在此!
1. 移除了闪烁标语文字的斜度，并且新增了一条闪烁标语，这条新的标语是一条随机的回声洞！
2. 将所有板块中的颜色大幅度调整为了`{DynamicResource ColorBrushX}`,现在看起来更舒服了!
3. 把不容易看清的文字前景色调整为白色(`#fff`), 同时加大了新闻板块的文字 Margin 。
4. 把欢迎页中的大字替换为了"欢迎使用缝合怪主页!",~~少了一个被骂的点!~~
5. 服务器板块移除了两个无用的版本启动按钮
取而代之的是两个从[MC找服网](mczfw.cn)找到的宝藏服务器!
还加入了提醒按钮以最大程度防止误会。
6. 淘汰了之前的 MostUsed 启动，使用`\current`代替
7. 之前的Wiki摘录有人提醒我说没有给出Wiki链接,其实在左边的常用网页里!
8. 移除了 Herobrine !

![awa](https://github.com/user-attachments/assets/0af09801-2767-4303-aa29-889212fba50a)
蓝色圈圈出的部分即为改动过的部分

谢谢您对于 FHG 主页的支持
祝您使用愉快!

**Reply by CreeperIsASpy** [reply]
@LTCatt 再看看?

**Reply by LTCatt** [reply]
不要加入服务器和版本选择板块，不管你怎么叠甲，服务器都是疑似第三方推广的东西，而 BE 又会让人误解 PCL 的用途。
UI 布局不够美观，这个我不太好提……

**Reply by CreeperIsASpy** [reply]
你怎么不去说简单主页误解呢?
我知道我永远比不过简单主页，我知道你想让简单主页变成唯一的主页
但是你好歹编出合理的理由啊！


服务器板块永远惹嫌疑，那么被嫌的是我，不是你！
UI我再怎么苦思冥想，还是比不过简单主页的，我就不说了。
请你给一个合理的理由吧！

**Reply by allMagicNB** [reply]
> 你怎么不去说简单主页误解呢?

因为他做的不像你做得那么大。

> 我知道我永远比不过简单主页，我知道你想让简单主页变成唯一的主页
> 但是你好歹编出合理的理由啊！

@MFn233 这边又有人说你垄断了。

> 服务器板块永远惹嫌疑，那么被嫌的是我，不是你！

那么，普通用户可不管那么多，他们只知道是开发者把他加进去的。这基本等于是龙猫在宣传服务器。

> UI我再怎么苦思冥想，还是比不过简单主页的，我就不说了。

那这边建议你别做了。

> 请你给一个合理的理由吧！

质量较低。

**Reply by MFn233** [reply]
@allMagicNB 你说“又有人”不太对，自始至终是同一个人……习惯了
chiling next将会是我继续操刀制作的炫技用的主页，虽然我没什么空维护就是。。。

**Reply by MFn233** [reply]
如果我有能力，我会让chiling next变成第二个“聚合”类型主页。

**Comment by MoYuan-CN** [comment]
几乎看完了整件事情之后想说的一些话（部分内容引用自 https://github.com/Hex-Dragon/PCL2/discussions/1952#discussioncomment-10147460 ，虽然说这条信息被隐藏了)

---
大约一年半以前，我也曾尝试过编写自定义主页，那时候我还在管理一个服务器。写着写着就发生了报错，那是我第一次接触 WPF 页面文件的代码。虽然以前接触过 Markdown、HTML 这种标记语言，但报错还是令我措手不及。找了半天论坛啊，什么的，在 PCL 的更多页面中找到了这个 GitHub 社区，然后就交了 #1339 反馈。那时候还不怎么知道反馈流程，就按照表单交了内容希望有人能帮到我。虽然后来 Issue 烂了，我也确认了是自定义主页文件里写错了点什么东西……
但无论如何，每个人都是从不会到会走过来的。也确实，现在互联网越来越不包容 “小白” 了。对于新手来讲，能够在社区/软件中创建属于自己的一份作品的机会越来越少了，越往后越困难（因为已经有别人做出的，还算不错的作品了）

---
以下内容是对上一楼部分回复的评论（并非针对某人，仅表达我对那句话/那件事情的看法）
> 那这边建议你别做了。

看到他人在某件事情上并没有做的太好的时候，没必要也不应该以这种语气让别人放弃。不要这样。对小白耐心一点，友好一点，并不是什么很难的事情。
> 服务器板块永远惹嫌疑，那么被嫌的是我，不是你！

尝试站在开发者的角度想一想，在主页上的内容，对于一般（小白/非开发者）用户来说，是否会认为是 PCL 官方制作的内容？答案是会。所以，服务器这种很容易引起舆论/较大主观判断是否优质的东西不适合放在容易被认为是官方内容的地方。
> 但是你好歹编出合理的理由啊！

其实没必要为了这一件事情而如此有情绪，可能确实你的主页（从开发者的角度看）有缺点，努力完善它或者找另一个自己擅长的方向再做尝试，或许你就能有所成就

**Reply by CreeperIsASpy** [reply]
我只想说一句话
`EventType="弹出窗口" EventData="关于&quot;服务器&quot;板块|本板块中的服务器皆选自mc找服网(mczfw.cn)\n并没有商业化,请勿误解。且关于列表中服务器的质量由我来担保。\n如果列表中服务器质量有问题，请到github仓库中向我反馈。" `
永远都会有人误解，不去看这条提醒，就无脑喷龙猫
就好像错误窗口里写"不要转发这个窗口的截图",还是有一堆人发截图......

**Reply by CreeperIsASpy** [reply]
> 那么，普通用户可不管那么多，他们只知道是开发者把他加进去的。这基本等于是龙猫在宣传服务器。

你基本就是在表达这个意思：普通用户都是脑残

> @MFn233 这边又有人说你垄断了。

挑衅，挑拨自定义主页作者之间的关系

我只能说，你也是个神人。
你当初是怎么当上collaborator的？

**Reply by wuliaodexiaoluo** [reply]
>挑衅，挑拨自定义主页作者之间的关系
>
>我只能说，你也是个神人。
>你当初是怎么当上collaborator的？

请注意控制语气以避免不必要的冲突，谢谢

**Reply by CreeperIsASpy** [reply]
> > 挑衅，挑拨自定义主页作者之间的关系
> > 我只能说，你也是个神人。
> > 你当初是怎么当上collaborator的？
> 
> 请注意控制语气以避免不必要的冲突，谢谢

人家不控制语气，凭什么我要控制？

**Reply by CreeperIsASpy** [reply]
龙猫在各方面都无比偏袒简单主页，无论你们有没有发现
我认为，这就是一种垄断
但是，这不是针对Mfn233的，mfn主页做的很好
是龙猫造成了垄断现象

### 我不是在骂MFn
### 我不是在骂MFn
### 我不是在骂MFn
### 我不是在骂MFn

**Reply by wuliaodexiaoluo** [reply]
>人家不控制语气，凭什么我要控制？

如果大家都像你这样，那社区早就吵成一锅粥了
你可能需要和 wf 谈下人生哲理

**Reply by MoYuan-CN** [reply]
正如我楼正文所说，服务器内容不适合放在主页上，这并不代表普通用户都是脑残
只是如果放上去了容易引起别的问题，例如：
- 某玩家对某服务器有负面看法，他发现这个服务器在 PCL 主页上，到处说 PCL 启动器不好。
- 某玩家看了主页后尝试去加入某服务器，但是没能成功，认为 PCL 不简单实用。
- 某服务器产生了不好的舆论，然后这个服务器在 PCL 主页上……

不建议也不允许自定义主页上出现服务器相关内容是综合考量过的，肯定不是一句话草草断定的

---

不要有太大情绪了，如果你确实找不到一个适合发泄的地方，来 QQ 1913532130 私信找我
现在暂时有空，我可以做一会你的树洞

**Reply by allMagicNB** [reply]
> 那这边建议你别做了。

这等于别做 UI，而非别做主页。
我看你意见似乎对我有点大？啥事都往坏的想。

**Reply by allMagicNB** [reply]
> > 那么，普通用户可不管那么多，他们只知道是开发者把他加进去的。这基本等于是龙猫在宣传服务器。
> 
> 你基本就是在表达这个意思：普通用户都是脑残

勿过度理解。我没这样说。

> > at MFn233 这边又有人说你垄断了。
> 
> 挑衅，挑拨自定义主页作者之间的关系
> 
> 我只能说，你也是个神人。
> 你当初是怎么当上collaborator的？

当初 MFn 发声明时，就是因为龙猫不再接受类似于简单主页的预设时发的。

**Comment by CreeperIsASpy** [comment]
### 大杂烩向我肯定做不过简单主页，我相信将来也没有人能做出来了

这样吧，我们来投个票：
- :eyes: 现在这样挺好，没必要转型
- :heart: 可以把wiki摘录单独做一个主页
- :rocket: 可以直接别做了

**Reply by MoYuan-CN** [reply]
我三个都选帮助你们做直接入口，最后统计的时候各 -1 即可
个人的意见是 :heart:

**Comment by wyc-26** [comment]
1. 服务器板块中的服务器
抛开疑似为广告的问题来看
这些服务器知名度都不是很高（指下面四个，CN2B2T 还行吧）
2. UI 问题
目前的 UI 似乎看上去有些生硬
可以考虑再加一点圆角
3. 格式问题
希望可以在中文与数字、字母之间加一个空格，美观度也许会提高一点
4. 背景图片过于刺眼（？
个人觉得可以换回 https://github.com/Hex-Dragon/PCL2/discussions/2528#discussioncomment-10301053 中的背景

这几个板块中我最满意的就是 Wiki 摘录了，希望可以坚持做下去
服务器板块还是考虑放弃吧

**Reply by CreeperIsASpy** [reply]
> 背景图片过于刺眼 

渐变色刺眼
白色也刺眼
我开始有点搞不懂了

**Reply by MoYuan-CN** [reply]
可能是你背景图片（PCL 设置）影响的？

**Reply by wyc-26** [reply]
我觉得深色的会好一点.jpg
也有可能是我啥软件都用深色主题然后习惯了（

**Reply by iPodtouch6557** [reply]
> > 背景图片过于刺眼
> 
> 渐变色刺眼 白色也刺眼 我开始有点搞不懂了

美观问题看着办吧，大家对于美观有不同见解

**Comment by Chiloven945** [comment]
这边的建议是您完全可以别做这个主页了呢

**Reply by MoYuan-CN** [reply]
按照你的想法去上边投票.png

**Comment by CreeperIsASpy** [comment]
mc找服网最顶上的没听说过
我摘的还不是付费置顶的

型
我现在就不搞了
我不搞PCL主页了

**Reply by CreeperIsASpy** [reply]
我花着开学之前最后一点时间来网上挨骂
何苦呢？
谁爱当黑奴谁干去吧

**Reply by Chiloven945** [reply]
> 我花着开学之前最后一点时间来网上挨骂
> 何苦呢？
> 谁爱当黑奴谁干去吧

请不要提及种族与肤色歧视的相关内容。

**Reply by iPodtouch6557** [reply]
初中生意气用事可以理解但没必要，至少作为大致的同龄人我不会这么干
希望未来的你有了明确灵感可以回来，也希望你的课业生活不会遇到各种サビ事情，开心就好啦

**Comment by WForst-Breeze** [comment]
请在沟通过程中保持冷静客观，我们双方都是无偿为 PCL 社区出力的，我们的根本目的是一致的，我们是同一战线的朋友，如若因为这个吵起来这是和我们的最初目的背道而驰的表现

提出的建议不是硬性要求，提建议的也不是甲方。对于双方，礼貌沟通是讨论的基础，这里欢迎思想的碰撞，但并不支持言语上的博弈，二者此消彼长，一旦讨论的主题从「怎么样」变成了「怎么就」，那么无论结果如何，这场讨论到头来也是无意义的

机会还有很多，有时候可以换换路子，没准会找到新的出口呢？

**Reply by MFn233** [reply]
同意。不要动不动就急了

**Reply by CreeperIsASpy** [reply]
对，但是歪曲事实、偷换概念是互联网人类赖以生存的本能

**Reply by MoYuan-CN** [reply]
问题是并没有人歪曲事实……
服务器板块就是那样，这可能只有你自己发表了 APP 作品然后别人投稿主观宣传性质的东西时你才能懂……

**Reply by iPodtouch6557** [reply]
> 同意。不要动不动就急了

可能是作业太多课业负担繁重吧……理解一下，事情恶劣了再说

**Comment by CreeperIsASpy** [comment]
希望github能够依靠自我净化能力抵御

**Comment by Pigeon0v0** [comment]
不能加入服务器相关内容是硬性规定，之前龙猫也曾明确表态过了

关于 BE 这块的内容占据了较大版面，不太合适，隔壁只是一个按钮

关于你之后这个态度，我说不好听点叫失败的次数不够多，再难听点就是被社会毒打少了。大家已经提醒你可以尝试换个方向，你还要硬碰硬，那就没啥办法了 ¯\\_(ツ)\_/¯

**Reply by allMagicNB** [reply]
¯\\\_(ツ)\_/¯

**Reply by CreeperIsASpy** [reply]
我花时间上网被人挨骂
我闲得慌啊

**Comment by CreeperIsASpy** [comment]
说我的主页做的不好的，都是我的错
没有一个人明确地指出哪里做的不好，除了 @hejiehao

**Reply by CreeperIsASpy** [reply]
@Chiloven945 我不知道我惹到你哪里了
我向你道歉
请你不要再追杀我了

**Reply by CreeperIsASpy** [reply]
我已经关闭这个帖子了
希望来年的寒假我们还能相见
请不要发表内容

**Reply by wyc-26** [reply]
> 请不要发表内容

你想让大家不发表内容也真是太困难了

> 说我的主页做的不好的，都是我的错 没有一个人明确地指出哪里做的不好，除了 @hejiehao

此外我对这句话有些想法
可以看看最上面的几条 comment
别人辛辛苦苦地写长文来想要让你的主页更好，你却不把这些反馈放在眼里，认为你的方向一直都是正确的

你硬要保留服务器这个板块的话，我尊重你的想法
Anyway，这样的话龙猫大概率是不会把你的主页放进主页预设里面的
你还是考虑在其他平台上宣传吧……

希望来年寒假你可以重新理顺一下思路
或许可以把 Wiki 摘录这部分单独投稿呢（
现在你的主要任务是搞好学习，也不多说了

**Comment by CreeperIsASpy** [comment]
能不能找到一个方法，把这个帖子彻底删除?

**Reply by CreeperIsASpy** [reply]
是这样的，我想用class，但是......
![image](https://github.com/user-attachments/assets/958bd92f-f6db-4716-9236-6afc20ff801c)
有什么解决方法吗?

**Reply by CreeperIsASpy** [reply]
@MFn233  @JingHai-Lingyun

**Reply by MFn233** [reply]
你unsub吧。眼不见心不烦。。。

**Reply by Deep-Dark-Forest** [reply]
我们虽然有删帖的权限，但是我们只能对那些对社区环境有危害的或者无意义的帖子进行删除，其他的我们也不可以删（

**Comment by MFn233** [comment]
所有社区宗旨：
0.保持友善 Remain Friendly

**Reply by MFn233** [reply]
第一，没有多少人说这个主页不好，若你对任何人有误解，请回看他们的回复
第二，这个主页的确需要按照要求整改，所有人都在陈述事实，没有针对到作者本人
第三，没有人在挑拨作者间的关系，没有任何两个作者间是绝对的敌对关系，除非其中一人先急了

**Reply by MFn233** [reply]
建议需要有舍有得，一切都需自己考虑。如果连筛选有效建议的能力都没有，光是急有什么用呢？谁手写的主页不是经过龙猫挑刺多少回之后不断改才上预设里的？

**Reply by MFn233** [reply]
我知道我说这些话你不愿意听，那就这样吧。有时候你不是标准，你就只能适应标准，因为社区的标准在大多数人来看是适合大多数人的。

**Comment by WForst-Breeze** [comment]
本 Discussion 已被关闭，**如无必要烦请各位不要再进行回复了** :D 这会对他人造成不必要的打扰

**Comment by CreeperIsASpy** [comment]
Wiki 摘录主页预览
![image](https://github.com/user-attachments/assets/b7f71753-e99e-4421-9b5b-24cbef63e76c)
另外......
![image](https://github.com/user-attachments/assets/56f9d527-b680-445b-ba22-c27316d2c75d)

累死我啦!

**Reply by CreeperIsASpy** [reply]
哦对了，这个排版是从信标的主页上抄的，得到他允许了:
![image](https://github.com/user-attachments/assets/3dd2bfe6-96cb-4ed9-87d5-4b34d40cccfc)

**Reply by allMagicNB** [reply]
？哥们你没给信标署名吧

**Reply by CreeperIsASpy** [reply]
有
但是我截图只截了一个板块
下面有信标的署名，协议链接，还有任何你会挑出来的刺......

**Reply by MoYuan-CN** [reply]
如果你在制作一个新的主页，请**新开一个 自定义主页 分区帖子**，而**不是在这下面回复**

**Reply by CreeperIsASpy** [reply]
呃，好。。。

**Reply by CreeperIsASpy** [reply]
我准备开发完成之后再新开一个帖子

**Comment by CreeperIsASpy** [comment]
- [x] 自动爬出图片和文字(由re改为BeautifulSoup)
- [x] 自动链接gitee更新仓库
- [ ] 自动改写Custom.xaml(定时运行)
- [ ] 自动获取仓库commit

**Reply by CreeperIsASpy** [reply]
目前可通过在启动器“联网更新”中填写以下链接来使用Wiki 摘录主页
https://gitee.com/planet_of_daniel/pcl2-homepage-fhg/raw/main/Custom.xaml

**Reply by CreeperIsASpy** [reply]
感谢 @Chiloven945 !他让我知道，在我身边还有人默默无闻地关注着我！
(他给我的每一条消息点了一个 Confused )
